mypy/plugins/dataclasses.py

```diff
@@ -125,7 +125,9 @@ def transform(self) -> None:
             'eq': _get_decorator_bool_argument(self._ctx, 'eq', True),
             'order': _get_decorator_bool_argument(self._ctx, 'order', False),
             'frozen': _get_decorator_bool_argument(self._ctx, 'frozen', False),
+            'slots': _get_decorator_bool_argument(self._ctx, 'slots', False),
         }
+        py_version = self._ctx.api.options.python_version
```

Comment:
I am going to use it with `unsafe_hash` later, so it is on the top level. See https://github.com/python/mypy/issues/11463